src/git.md

```diff
@@ -176,6 +176,8 @@ There is a workaround in [the issue][#77620-workaround].
 [#77620]: https://github.com/rust-lang/rust/issues/77620
 [#77620-workaround]: https://github.com/rust-lang/rust/issues/77620#issuecomment-705228229
 
+(Note that as of <!-- date-check --> Sept 2022 `miri` is a subtree and not a submodule.)
```

Comment:
I don't think we have to do the date check as the state isn't likely to be changed in the near future.

Reply:
done